sklearn/linear_model/logistic.py

```diff
@@ -1725,3 +1725,30 @@ def fit(self, X, y, sample_weight=None):
 
         self.C_ = np.asarray(self.C_)
         return self
+
+    def score(self, X, y, sample_weight=None):
+        """Returns the mean accuracy on given test data and labels.
+
+        Due to a bug in scikit-learn this method always returns mean accuracy.
+        This will be fixed in version 0.20. For now you can externally call the
```

Comment:
@jnothman Thank you for your comment. It is my first contribution to the community and I will try my best to contribute more in the future.